extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -57,6 +63,9 @@ const SliderType = {
   EMOJI: 'emoji',
 };
 
+const SLIDER_HINT_DURATION_MS = 1500;
+const ANIMATION_DELAY_MS = 500;
```

Comment:
nit: Naming these could be made consistent:
HINT_ANIMATION_DURATION_MS
HINT_ANIMATION_DELAY_MS